src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+                </p>
+              </div>
+              <v-row v-if="item.expandedInfo2" no-gutters class="pt-7">
+                <span v-html="item.expandedInfo2">{{ item.expandedInfo2 }}</span>
+              </v-row>
+              <v-row v-if="item.expandedInfo3" no-gutters class="pt-7">
+                <span v-html="item.expandedInfo3">{{ item.expandedInfo3 }}</span>
+              </v-row>
+              <v-row v-if="item.expandedInfo4" no-gutters class="pt-7">
+                <span v-html="item.expandedInfo4">{{ item.expandedInfo4 }}</span>
+              </v-row>
+              <v-row v-if="[ITEM_TYPES.EXACT_MATCH, ITEM_TYPES.SIMILAR_MATCH].includes(item.info)"
+                     class="pt-3"
+                     no-gutters>
+                <v-btn class="outlined pa-0 tips-btn"
+                       :ripple="false"
+                       @click="item.expandExtraInfo = !item.expandExtraInfo">
+                  <span>
+                    <span v-if="!item.expandExtraInfo" right>View </span>
+                    <span v-else right>Hide </span>
+                    tips for how to create a unique name
+                  </span>
+                </v-btn>
+              </v-row>
+              <v-row v-if="item.expandExtraInfo" no-gutters class="pt-3">
+                <v-col>
+                  <v-row no-gutters>
+                    <b>Use words that will set your name apart, such as:</b>
+                  </v-row>
+                  <v-row no-gutters>
+                    <ul>
+                      <li>a person's full Name</li>
+                      <li>a geographical location</li>
+                      <li>a coined, made-up word, or</li>
+                      <li>an acronym</li>
+                    </ul>
+                  </v-row>
+                </v-col>
+              </v-row>
+            </v-col>
+          </v-row>
+        </v-col>
+      </v-row>
+    </template>
+  </v-data-table>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+
+import QuickSearchNames from '@/components/new-request/name-check/quick-search-names.vue'
+import { NameCheckItemIF } from '@/interfaces/name-check-interfaces'
+import { NameCheckErrorType, NameCheckItemType } from '@/enums'
+
+@Component({
+  components: { QuickSearchNames }
+})
+export default class NameCheckConflicts extends Vue {
+  @Prop()
+  private readonly items: Array<NameCheckItemIF>
+
+  @Emit() private clearError (value: string) { }
+  @Emit() private retry (value: string) { }
+
+  private expanded: Array<NameCheckItemIF> = []
+  private headers = [
+    { text: 'Icon', value: 'icon' },
+    { text: 'Problem', value: 'problem' },
+    { text: 'Words', value: 'words' },
+    { text: '', value: 'data-table-expand' }
+  ]
+
+  get ERROR_TYPES () {
+    return NameCheckErrorType
+  }
+  get ITEM_TYPES () {
+    return NameCheckItemType
+  }
+
+  @Watch('expanded')
+  private triggerChange () { }
```

Comment:
Obsolete?

Reply:
no I need it otherwise the change doesn't trigger after it's been mounted

Reply:
Is this something that `{ immediate: true }` could solve?

If you keep this (with an empty body), DEFINITELY add a comment, otherwise some over-zealous programmer might delete this.

Reply:
Okay good point haha

Reply:
Okay I am adding a comment. I'm not sure why it doesn't act reactive otherwise